.gitignore

```diff
@@ -132,3 +115,7 @@ _site
 /docs/.bundle/
 /docs/vendor/
 /lib/bundler/man/
+
+# Benchmarks
+*.csv
```

Comment:
I think this is too strict, we could have in the future a `csv` in the tests for example